modules/ROOT/pages/connected-apps-overview.adoc

```diff
@@ -13,9 +13,46 @@ The Connected Apps feature enables you to use secure authentication protocols an
 
 Anypoint Platform supports OAuth 2.0 and OpenID Connect to authorize apps to access data within Anypoint Platform.
 OAuth (Open Authorization) is an open protocol that provides secure API authorization from applications in a simple and standardized way.
-OAuth can authorize access to resources without revealing user credentials to apps.
+OAuth 2.0 can authorize access to resources without revealing user credentials to apps.
 OpenID Connect identifies the end user and obtains information to pass to OAuth 2.0 connected apps.
 
+
+
+The Connected Apps feature implements OAuth 2.0 and OpenID Connect, or _OIDC_, for authorizing access to data. OAuth 2.0 and OpenID Connect flows grant permission to services and users to access the Anypoint APIs.
+
+=== OAuth 2.0
+
+The OAuth 2.0 authorization protocol depends on several elements to function. 
+
+Client::
+The resource that requests data on behalf of a user or service.
+
+Authorization server::
+The server that gives access tokens to the client once it does authorization.
+
+Resource server::
+The system that contains the requested data.
+
+Access token:: 
+A "key" needed to access data in the resource server. 
```

Comment:
`Access Keys` are [another method of authz](https://swagger.io/docs/specification/authentication/api-keys/) that predates OAuth & is generally regarded as not best practice. Can you find a different way to describe tokens that differentiates them from keys? (The RFC is helpful here too)